com.usebruno.Bruno.yml

```diff
@@ -0,0 +1,54 @@
+id: com.usebruno.Bruno
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+command: bruno
+finish-args:
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+modules:
+  - name: unappimage
+    buildsystem: simple
+    build-commands:
+      - make -C squashfs-tools install INSTALL_DIR=/app/bin
+    sources:
+      - type: git
+        url: https://github.com/refi64/unappimage
+        commit: d7f86f2a0d7ec3a69211125207d5f127386b849a
+
+  - name: bruno
+    buildsystem: simple
+    sources:
+      - type: script
+        dest-filename: bruno
+        commands:
+          - zypak-wrapper.sh /app/extra/bruno/bruno --ozone-platform-hint=auto --enable-features=WaylandWindowDecorations "$@"
+
+      - type: extra-data
+        url: https://github.com/usebruno/bruno/releases/download/v1.5.1/bruno_1.5.1_x86_64_linux.AppImage
```

Comment:
so how does this work on aarch64?

Reply:
As of now we want to support `x64`, we plan to add support for `aarch64` later.

Reply:
you still build an aarch64 flatpak here.

Reply:
@hfiguiere Sorry, Can you expand. Are you saying the we should be able to add config for building `aarch64` in the future in the same manifest ?

Reply:
https://docs.flathub.org/docs/for-app-authors/maintenance#limiting-the-set-of-architectures-to-build-on

Reply:
Thanks @bbhtt 
I have added `flathub.json` file